Gemfile

```diff
@@ -5,3 +5,5 @@ gem 'puppet', puppetversion
 gem 'puppetlabs_spec_helper', '>= 0.1.0'
 gem 'puppet-lint', '>= 1.0.0'
 gem 'facter', '>= 1.7.0'
+gem 'rspec-puppet', '~>1.0'
+gem 'rspec', '~>2.0'
```

Comment:
For consistency between modules, could you change the rspec line to match this

https://github.com/ghoneycutt/puppet-module-pam/blob/master/Gemfile#L14-17